### PR TITLE
Added Support for Crew Tracking and Orbiters

### PR DESCRIPTION
**Pull request by JRascagneres**
Includes Orbiter Tracking --> Crewed and Uncrewed
Space Station Tracking

### CODE REVIEW

File: api/admin.py [hunk 1]
```diff
@@ -168,3 +172,33 @@ class VidAdmin(admin.ModelAdmin):
 class InfoAdmin(admin.ModelAdmin):
     icon = '<i class="material-icons">link</i>'
     list_display = ('info_url', 'launch')
+
+
+@admin.register(models.AstronautStatus)
+class AstronautStatusAdmin(admin.ModelAdmin):
```
Comment (ItsCalebJones) [inline api/admin.py]: This was moved to configurations, right?

File: api/admin.py [hunk 2]
```diff
@@ -168,3 +172,33 @@ class VidAdmin(admin.ModelAdmin):
...
+    list_display = ('name',)
+
+
+@admin.register(models.SpaceStation)
+class SpaceStationAdmin(admin.ModelAdmin):
+    list_display = ('name', )
+
+
+@admin.register(models.OrbiterStatus)
+class OrbiterStatusAdmin(admin.ModelAdmin):
```
Comment (ItsCalebJones) [inline api/admin.py]: This was moved to configurations, right?

File: api/admin.py (hunk 1 above)
Comment (JRascagneres) [reply api/admin.py]: Yes woops! Looks like I left the admin settings behind!

File: api/admin.py (hunk 2 above)
Comment (JRascagneres) [reply api/admin.py]: Yes woops! Looks like I left the admin settings behind!

**Pull request merged**